development.md

```diff
@@ -0,0 +1,70 @@
+# VMWare
```

Comment:
This all makes sense to me and I was able to follow it to generate the modules locally. I did notice that the generated modules didn't pass black linting locally. Do we want to include instructions to reformat the modules with black after they are generated?

Reply:
Much better!